nautobot/extras/views.py

```diff
@@ -1009,6 +1009,8 @@ def post(self, request, class_path=None, slug=None):
             messages.error(request, "Unable to run or schedule job: Job is not presently installed.")
         elif not job_model.enabled:
             messages.error(request, "Unable to run or schedule job: Job is not enabled to be run.")
+        elif job_model.has_sensitive_variables and request.POST["_schedule_type"] != JobExecutionType.TYPE_IMMEDIATELY:
+            messages.error(request, "Jobs with sensitive variables cannot be scheduled for later execution.")
```

Comment:
Let's make this more consistent with the other errors in this view:

```suggestion
            messages.error(request, "Unable to schedule job: Job has sensitive input variables.")
```